views/newChoral.pug

```diff
@@ -15,18 +15,22 @@ block content
     .form-group
       label(for='choralSampleRateInput') Sample Rate (seconds)
       .input-group
-        input#choralSampleRateInput.form-control(name='sampleRate' type='text', placeholder='5')
+        input#choralSampleRateInput.form-control(name='sampleRate' type='number', value='5')
         .input-group-addon.w3-blue seconds (>= 1)
-    .form-group
-      label(for='choralFuncInput') Function
-      textarea#choralFuncInput.form-control(name='func' type='text' placeholder='function(children, done){ done({ val: Math.random() * 50 }); }')
-      
-    if defaultFuncs.length > 0
+    #choralFuncInput
       .form-group
+        label(for='choralFuncInput') Function
+        textarea.form-control(name='func' type='text' )
+          | function(children, done){
+          |     done({ val: Math.random() * 50 });
+          | }
+
+    if defaultFuncs.length > 0
+      .form-group.inline-select
         label(for='choralDefaultFuncInput') Quick Function
-          select(name='defaultFunc' type='text').form-control#choralDefaultFuncInput
+          select(name='defaultFunc').form-control.add-child-input#choralDefaultFuncInput
```

Comment:
I am not a big fan of this
![image](https://user-images.githubusercontent.com/8840965/28629649-e17afd6e-71dc-11e7-936e-cebe5bde5954.png)